hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/KeyValueContainerUtil.java

```diff
@@ -114,10 +114,8 @@ public static void createContainerMetaData(
               "Unrecognized schema version for container: " + schemaVersion);
     }
 
-    ReferenceCountedDB db =
-        new ReferenceCountedDB(store, dbFile.getAbsolutePath());
```

Comment:
we should still have a refcounted DB here ?

Reply:
@mukul1987 Yes, actually we still have it to be a referenced DB inside the following call BlockUtils.addDB().
After we introduced new DatanodeStore definitions, we will have the logic to check whether we want a referenced DB or a RawDB inside BlockUtils.addDB() according to the schema version.